### ISSUE TITLE
Remediation for Implicit PendingIntent Vulnerability

### ISSUE THREAD

**Issue by eltohamy** [facebook/facebook-android-sdk#976]
- [Yes ] I've updated to the latest released version of the SDK
- 
```java
Intent intent = new Intent(context, CurrentAccessTokenExpirationBroadcastReceiver.class);
        intent.setAction("com.facebook.sdk.ACTION_CURRENT_ACCESS_TOKEN_CHANGED");
        PendingIntent pendingIntent = PendingIntent.getBroadcast(context, 0, intent, 0);
        try {
          alarmManager.set(1, accessToken.getExpires().getTime(), pendingIntent);
          return;
        } catch (Exception exception) {
          return;
        } ```

All PendingIntents usage need to be changed as per Google recommendation in this link https://support.google.com/faqs/answer/10437428

**Comment by Kurnosov** [comment]
Cause crash on FB login on Android 12 (S+)

**Comment by linmx0130** [comment]
@Kurnosov Thanks for pointing out. The bug fix will be released in FBSDK v11.3.0 in early September. Also thanks @eltohamy  for raising this issue.

**Comment by eltohamy** [comment]
@linmx0130 Welcome, Thank you for your support.
